ansible/roles/test/tasks/vxlan-decap.yml

```diff
@@ -31,14 +31,24 @@
       vars:
         supervisor_host: "{{ ptf_host }}"
 
+    - name: Start arpresponder
+      supervisorctl: state=started name=arp_responder
```

Comment:
Suggest use restarted, sometimes the service could have been running because previous test didn't clean up. I noticed you do clean up in always block. I learned that if something went wrong in always block before reaching service stop. The play book could exit and leave the service running.